### PR TITLE
Add test case for CommandLineShim class.

### PR DESCRIPTION
**Pull request by fuji-151a**
There is no test class in CommandLineShim.
So, I created CommnadLineShimTest class and I wrote the test case. 
Please check this pull request.

Thanks.

### CODE REVIEW

File: storm/src/test/storm/mesos/shims/CommandLineShimTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                = "export STORM_SUPERVISOR_LOG_FILE=" +
+                topologyId +
+                "-supervisor.log " +
+                "&& cp storm.yaml storm-mesos*/conf " +
+                "&& cd storm-mesos* " +
+                "&& bin/storm supervisor storm.mesos.MesosSupervisor" +
+                extraConfig;
+        assertThat(actualCmd, is(expectedCmd));
+    }
+}
```
Comment (fuji-151a) [inline storm/src/test/storm/mesos/shims/CommandLineShimTest.java]: OK. I'll try to rewrite code to put a newline end of the file.

File: storm/src/test/storm/mesos/shims/CommandLineShimTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+public final class CommandLineShimTest {
+
+    @Test
+    public void getCommandLine() {
+        String extraConfig = " -c storm.local.dir=/tmp";
+        String topologyId = "sample-id";
+        CommandLineShim target = new CommandLineShim(extraConfig);
+        String actualCmd = target.getCommandLine(topologyId);
+        String expectedCmd
+                = "export STORM_SUPERVISOR_LOG_FILE=" +
```
Comment (fuji-151a) [inline storm/src/test/storm/mesos/shims/CommandLineShimTest.java]: OK. I'll try to move the `=` to the previous line.

**Comment by fuji-151a** [comment]
@erikdw Thank you for review.
I rewrote code based on your review.
Please check again.

**Comment by erikdw** [comment]
@fuji-151a : great, thank you for making those changes, and for your contributions to the tests!

**Pull request merged**